.github/workflows/node.js.yml

```diff
@@ -16,7 +16,8 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [ 10.x ]
+        # TODO: hoist to 10.x when we bump the version supported in #5931
+        node-version: [ 8.x ]
```

Comment:
Is this the action that tests across multiple versions? If so, why not list 8.x, 10.x, and 12.x?
Perhaps even [`14.x`](https://github.com/peterbe/minimalcss/blob/e432a59bb085c964dfc354f9327975f65ff78767/.github/workflows/node.js.yml#L19).

Reply:
The short version is: I'm going to fix this in a subsequent PR, but I wanted to make the Node versions consistent between workflows now. Though it occurs to me now that the _release_ workflow should never do more than one version. I'll fix that now.

I think the plan here was to have it test multiple versions of Node once we had dropped Travis (which is also testing one more than one Node, I think). I just figured I should actually test with the same version as we specify as the minimum Node version.